src/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
we have it in root in all other apps

Reply:
Thought it made sense since the ESLint only applies to JS in `src/`. Still would like to move it to the root?

Reply:
Well it should follow the app-tutorial as good as possible, so everything is the same in all apps as much as possible